configs/components/echam/echam.yaml

```diff
@@ -785,6 +805,9 @@ namelist_changes:
                 parctl:
                         nproca: "${nproca}"
                         nprocb: "${nprocb}"
+                submodelctl:
+                        lgfw: "${lgfw}"
+                        lupdate_orog: "${lupdate_orography}"
```

Comment:
Is this bound to the version of echam? I'm asking because if it is not, other versions of echam might break when you add this new variables to the namelist.